### ISSUE TITLE
conflicting documentations about bootnode

### ISSUE THREAD

**Issue by bap2pecs** [ethereum/go-ethereum#26053]
In README, it says:
> Note: You could also use a full-fledged geth node as a bootnode, but it's the less recommended way.

However, when I run the `bootnode` tool, the log prints
> We recommend using a regular node as bootstrap node for production deployments.

I am not sure which one is accurate but one of them should be updated.

**Comment by MariusVanDerWijden** [comment]
I think the recommended way is to create a node key with the `bootnode` command and then run a full geth node as a bootnode

**Comment by bap2pecs** [comment]
thanks! @MariusVanDerWijden is `--nodekey` the option to run a full geth node as a bootnode?

**Comment by samba9274** [comment]
According to [this](https://ethereum.stackexchange.com/a/79278) answer on stackexchange in reference to [documentation](https://geth.ethereum.org/docs/interface/private-network#setting-up-networking), while running `geth`, you will have to provide `--nat` flag with host IP address.

In [documentation](https://geth.ethereum.org/docs/interface/command-line-options), there are more command-line options under the heading `NETWORKING OPTIONS` which will help.

**Comment by MariusVanDerWijden** [comment]
@bap2pecs yes thats how you specify the node key for the bootnode. Sorry for dropping the ball on this one. Will close for now

**Comment by MariusVanDerWijden** [comment]
Also fixing the documentation here: https://github.com/ethereum/go-ethereum/pull/30812
